### ISSUE TITLE
Issues with adding shapes

### ISSUE THREAD

**Issue by CentroneF** [gitbrent/PptxGenJS#660]
Hi,

I am not able to add any shape, specifically a rectangle to the slide. I tested on all the demos and actually is not working anywhere.
Here is my code:

```
    const pptx = new PptxGenJS();
    pptx.setLayout('LAYOUT_WIDE');
    pptx.defineSlideMaster({ title: 'MASTER_SLIDE',  bkgd:  'FFFFFF', objects: [], });
    const slide = pptx.addNewSlide('MASTER_SLIDE');
    // This is visible
    slide.addText('How To Create PowerPoint Presentations with JavaScript', { x:0.5, y:0.7, fontSize: 18 });
    //This is not visible
    slide.addShape(pptx.shapes.RECTANGLE, { x: 34.0, y: 80.0, w: 213.0, h: 394.0, fill: { type: 'solid', color: 'C74634'} });
    pptx.save('test');
```

I am using the angular version and I am importing js files like suggested in the documentation. 
Am I doing anything wrong or what is the issue? Do I need to import the shapes library and how?

Thank you
Francesco

**Comment by gitbrent** [comment]
Duplicate of Issue #656 

This will be fixed in v3.1.1 soon.

In the meantime, you can copy `types/index.d.ts` into your project's node_modules/pptxgenjs folder.

**Comment by CentroneF** [comment]
Hi, 

I don't think this is duplicate. I don't see any error in the compiler. The Rectangle is not showing in the PPT file. 

I am attaching the ppt file as result of the code above.
[test (2).pptx](https://github.com/gitbrent/PptxGenJS/files/4116327/test.2.pptx)

Francesco
